CMakeLists.txt

```diff
@@ -141,28 +141,40 @@ if(${CMAKE_COMPILER_IS_GNUCXX})
   message(STATUS "CMAKE_C_FLAGS = ${CMAKE_C_FLAGS}")
 endif(${CMAKE_COMPILER_IS_GNUCXX})
 
+message(STATUS "CMAKE_SYSTEM_NAME = ${CMAKE_SYSTEM_NAME}")
 if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-  if(${CMAKE_CXX_COMPILER_ID} MATCHES Clang)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC  -funroll-loops -Wall -Wno-unused-local-typedef -Wno-unknown-pragmas  ")
-    set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -fPIC -funroll-loops -Wall -Wno-unknown-pragmas  ")
-    set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined")
-    message("additional default options were set for clang compiler")
-    message("CMAKE_CXX_FLAGS = ${CMAKE_CXX_FLAGS}")
-    message("CMAKE_C_FLAGS = ${CMAKE_C_FLAGS}")
-  endif(${CMAKE_CXX_COMPILER_ID} MATCHES Clang)
-  if(${CMAKE_CXX_COMPILER_ID} MATCHES Intel)
-    if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 18.0)
-      # pybind requires min. version 17, but we need at least 18:
-      message( "DEPRECATED: detected compiler as Intel " ${CMAKE_CXX_COMPILER_VERSION} )
-      message( FATAL_ERROR "Please use Version 18 or greater")
-    endif()
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC  -funroll-loops -lpthread -wd654 -wd10010  ")
-    set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -fPIC -funroll-loops -lpthread -wd654 -wd10010  ")
-    set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined")
-    message("additional default options were set for intel compiler")
-    message("CMAKE_CXX_FLAGS = ${CMAKE_CXX_FLAGS}")
-    message("CMAKE_C_FLAGS = ${CMAKE_C_FLAGS}")
-  endif(${CMAKE_CXX_COMPILER_ID} MATCHES Intel)
+  if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  	  if(${CMAKE_CXX_COMPILER_ID} MATCHES Clang)
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}  -funroll-loops -Wall -Wno-unused-local-typedef -Wno-unknown-pragmas  ")
+		set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -funroll-loops -Wall -Wno-unknown-pragmas  ")
+		set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined")
+		message("additional default options were set for clang compiler")
+		message("CMAKE_CXX_FLAGS = ${CMAKE_CXX_FLAGS}")
+		message("CMAKE_C_FLAGS = ${CMAKE_C_FLAGS}")
+	  endif(${CMAKE_CXX_COMPILER_ID} MATCHES Clang)
+  else(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
I am pretty sure it will never never enter here. (Lines 146 + 155 = False)

Reply:
Sorry, copy paste